models/Res2NetSE50.py

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/python
+# -*- encoding: utf-8 -*-
+
+import torch
+import torch.nn as nn
+import torchaudio
+
+from models.Res2NetBlocks import SEBottle2neck
+from utils import PreEmphasis
+
+
+class Res2NetSE(nn.Module):
+    def __init__(self, block, layers, num_filters, nOut, encoder_type='SAP', attention_dim=128, baseWidth=26, scale=4, n_mels=40, log_input=True, **kwargs):
+        super(Res2NetSE, self).__init__()
+
+        print('Embedding size is %d, encoder %s.'%(nOut, encoder_type))
+
+        self.inplanes = num_filters[0]
+        self.encoder_type = encoder_type
+        self.n_mels = n_mels
+        self.log_input = log_input
+        self.baseWidth = baseWidth
+        self.scale = scale
+
+        self.conv1 = nn.Conv2d(1, num_filters[0], kernel_size=3, stride=1, padding=1)
+        self.relu = nn.ReLU(inplace=True)
+        self.bn1 = nn.BatchNorm2d(num_filters[0])
+
+        self.layer1 = self._make_layer(block, num_filters[0], layers[0])
+        self.layer2 = self._make_layer(block, num_filters[1], layers[1], stride=(2, 2))
+        self.layer3 = self._make_layer(block, num_filters[2], layers[2], stride=(2, 2))
+        self.layer4 = self._make_layer(block, num_filters[3], layers[3], stride=(2, 2))
+
+        self.instancenorm = nn.InstanceNorm1d(n_mels)
```

Comment:
I msft res2net [paper](https://arxiv.org/pdf/2007.02480.pdf) for speaker verification they use mean-only nomalization.